test/unit/models/classes/Security/DataAccess/Repository/CacheKeyChainRepositoryTest.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+/**
+ * Created by PhpStorm.
+ * User: bartlomiejmarszal
+ * Date: 2020-08-11
+ * Time: 13:50
+ */
+
+namespace oat\taoLti\test\unit\models\classes\Security\DataAccess\Repository;
+
+use oat\generis\test\MockObject;
+use oat\generis\test\TestCase;
+use oat\oatbox\cache\SimpleCache;
+use oat\tao\model\security\Business\Domain\Key\Key;
+use oat\tao\model\security\Business\Domain\Key\KeyChain;
+use oat\tao\model\security\Business\Domain\Key\KeyChainQuery;
+use oat\taoLti\models\classes\Platform\Service\KeyChainGenerator;
+use oat\taoLti\models\classes\Security\DataAccess\Repository\CachedPlatformKeyChainRepository;
+use oat\taoLti\models\classes\Security\DataAccess\Repository\PlatformKeyChainRepository;
+
+class CacheKeyChainRepositoryTest extends TestCase
+{
+    public const IDENTIFIER = 'id';
+    /** @var CachedPlatformKeyChainRepository */
+    private $subject;
```

Comment:
```suggestion
    public const IDENTIFIER = 'id';
    
    /** @var CachedPlatformKeyChainRepository */
    private $subject;
```

According to our CR guideline.